contracts/pm/ERC20TicketBroker.sol

```diff
@@ -21,37 +21,45 @@ contract ERC20TicketBroker is TicketBroker {
         token = ERC20(_token);
     }
 
+    // For ETH
+    function fundDeposit() external payable {
+        require(false, "ETH funding not supported. Please use an ERC20 token instead.");
+    }
+
+    function fundPenaltyEscrow() external payable {
+        require(false, "ETH funding not supported. Please use an ERC20 token instead.");
+    }
+
+    // For ERC20
+    function fundDeposit(uint256 _amount) external {
+        _fundDeposit(_amount);
+    }
+
+    function fundPenaltyEscrow(uint256 _amount) external {
+        _fundPenaltyEscrow(_amount);
+    }
+
     function fundAndApproveSigners(
         uint256 _depositAmount,
         uint256 _penaltyEscrowAmount,
         address[] _signers
     )
         external
-        processDeposit(msg.sender, _depositAmount)
-        processPenaltyEscrow(msg.sender, _penaltyEscrowAmount)
+        payable
```

Comment:
Add `require(msg.value == 0, ERROR_MSG)` here to reject calls that include ETH?

Reply:
👍